logging.js

```diff
@@ -171,6 +173,7 @@ export class LoggingClient {
 	constructor(appId, logger, opts) {
 		this._appId = appId;
 		this._logger = logger;
+		this._logHistory = [];
```

Comment:
Minor: Found this a bit confusing at the beginning, compared to something like `_logTimestamps`, but could just be me. You can dig into the code to see we don't actually keep a history of the full log sent.